circle.yml

```diff
@@ -10,10 +10,10 @@ test:
   pre:
     - mozdownload --version latest --destination firefox.tar.bz2
     - mozinstall firefox.tar.bz2
-    - wget "https://github.com/mozilla/geckodriver/releases/download/v0.9.0/geckodriver-v0.9.0-linux64.tar.gz"
-    - gunzip -c geckodriver-v0.9.0-linux64.tar.gz | tar xopf -
+    - wget -O geckodriver.tar.gz https://github.com/mozilla/geckodriver/releases/download/v0.10.0/geckodriver-v0.10.0-linux64.tar.gz
+    - gunzip -c geckodriver.tar.gz | tar xopf -
     - chmod +x geckodriver
-    - sudo mv geckodriver /home/ubuntu/bin/wires
+    - sudo mv geckodriver /home/ubuntu/bin
```

Comment:
It was renamed from `wires` to `geckodriver` and Selenium 3.0.0 looks for the new name.